src/order/rel_classes.lean

```diff
@@ -74,6 +74,14 @@ protected theorem is_asymm.is_irrefl [is_asymm α r] : is_irrefl α r :=
 protected theorem is_total.is_trichotomous (r) [is_total α r] : is_trichotomous α r :=
 ⟨λ a b, or.left_comm.1 (or.inr $ total_of r a b)⟩
 
+instance is_refl_of_irrefl {α : Type u} (r : α → α → Prop) [is_irrefl α r] :
+  is_refl α (λ x y, ¬ r y x) :=
```

Comment:
```suggestion
  is_refl α (λ x y, ¬ r x y) :=
```
Or if you really needed the swapped version, put `swap` in the name

Reply:
If you really wanted, you could use
```suggestion
  is_refl α (rᶜ) :=
```
(With the right imports) but that might be less convenient

Reply:
If you really wanted, you could use
```suggestion
  is_refl α (rᶜ) :=
```
(With the right imports) but that might be less convenient

Reply:
Any idea what the right imports are? I can't find this anywhere.

Reply:
The instance is `pi.has_compl`

Reply:
I find it a bit inconvenient to use `compl`, so I'll leave them as is.